src/statement-list/statement_list_parse_tree.h

```diff
@@ -85,129 +65,114 @@ class statement_list_parse_treet
     /// Sets the title of the network to a specific value.
     /// \param value: New title. Use an empty string to indicate that there is
     ///   no title.
-    void set_title(std::string &value)
-    {
-      if(value.empty())
-        title.reset();
-      else
-        title = value;
-    }
-
+    void set_title(const std::string &value);
     /// Adds an instruction to the network.
-    void add_instruction(instructiont &inst)
-    {
-      instructions.push_back(inst);
-    }
+    void add_instruction(const instructiont &inst);
 
     /// Create the network with a specific \p title.
-    explicit networkt(std::string &title)
-    {
-      set_title(title);
-    }
-
-    networkt()
-    {
-    }
+    explicit networkt(const std::string &title);
 
-    /// Prints the network in a human-readable form to the given output stream.
-    void output(std::ostream &out) const;
+    networkt() = default;
   };
   using networkst = std::list<networkt>;
 
-  /// Structure for a simple function in Statement List. Includes fields for
-  /// its name, version, variable declarations and networks.
-  struct functiont
+  /// Base element of all modules in Siemens TIA.
```

Comment:
What is TIA?

Reply:
The IDE by Siemens for programming their PLCs. Added a comment that describes the abbrevation.